package.json

```diff
@@ -30,12 +30,18 @@
   "license": "ISC",
   "devDependencies": {
     "@parcel/transformer-inline-string": "^2.5.0",
+    "assert": "^2.0.0",
     "buffer": "^6.0.3",
+    "crypto-browserify": "^3.12.0",
     "events": "^3.3.0",
     "next": "^12.1.6",
     "parcel": "^2.5.0",
     "patch-package": "^6.4.7",
-    "process": "^0.11.10"
+    "process": "^0.11.10",
+    "querystring-es3": "^0.2.1",
+    "stream-browserify": "^3.0.0",
+    "stream-http": "^3.2.0",
+    "url": "^0.11.0"
```

Comment:
I don't recall adding these, may be a helpful feature of parcel or IDE. I don't care enough to filter through them and get rid of what we don't need.